activerecord/lib/arel/select_manager.rb

```diff
@@ -131,7 +131,7 @@ def project(*projections)
       # FIXME: converting these to SQLLiterals is probably not good, but
       # rails tests require it.
       @ctx.projections.concat projections.map { |x|
-        STRING_OR_SYMBOL_CLASS.include?(x.class) ? Nodes::SqlLiteral.new(x.to_s) : x
+        STRING_OR_SYMBOL_CLASS.include?(x.class) ? Nodes::SqlLiteral.new(x.to_s, retryable: true) : x
```

Comment:
These sound user-supplied, as does the below

Reply:
Fixed 👍